back/config/locales/en.yml

```diff
@@ -91,6 +91,9 @@ en:
     services_description: Public services
     other: Other
     other_description: Other
+    quality_of_life: Quality of life
+    service_delivery: Service delivery
+    governance_and_trust: Governance and trust
```

Comment:
I think in Rob's designs, we use "Governance & trust" rather than "and" - super minor though, and something we can tweak later if needed :)

Reply:
Hmm, I'd always rather use the word instead of an &

Reply:
Ok with me! Just wanted to mention it. Let's go with "and" :heavy_check_mark: